scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/ParquetAvroIO.scala

```diff
@@ -288,6 +302,14 @@ object ParquetAvroIO {
     tempDirectory: String = WriteParam.DefaultTempDirectory,
     filenamePolicySupplier: FilenamePolicySupplier = WriteParam.DefaultFilenamePolicySupplier
   )
+
+  private[avro] def getDataSupplier(
+    key: String,
+    conf: Configuration
+  ): Option[Class[_ <: AvroDataSupplier]] =
+    Option(conf.getClass(key, null)).map { case _ =>
```

Comment:
`Configuration` doesn't have a `.containsKey` type method, thus this awkward workaround 😅